project-communications/academic-updates/researchfish-updates-2021.md

```diff
@@ -0,0 +1,899 @@
+**Categorisation of impact**	Societal
+**Formally Governed**	Yes
+ 
+**Collaboration Title**	Expert contributions from CRI 
+Partner
+Organisation Name	Interdisciplinary Research Centre
+Contributed Financially	No
+In-kind contribution	Yes
+In-kind contribution currency	EUR Euro
+Give an estimate of the in-kind value.	2000
+Contributions Made	We have shared our research with CRI, and provided them with a full presentation of the project and its aims by the Principle Investigator Kirstie Whitaker. We have also provided developmental internship opportunities for CRI students: doctoral students Katharina Kloppenborg and Sowmya Raja, who have both been offered substantial support and mentorship from the research team.   
+Partner Contributions	CRI has provided us with pro bono support from two highly skilled students at a doctorl level to run user-testing and develop the platform. They have also provided opportunities to disseminate the project's work and values which has been mutually beneficial. 
+Year Commenced	2020
+Year Ended	Still Active
+URL	
+Resultant Outcomes	Openly published user testing results with diverse autistic participants, as well as documentation for how other researchers could run them. https://github.com/alan-turing-institute/Autistica CitizenScience/tree/master/community-recommendatio ns/user-tests.   Presentation for CRI on "Participatory citizen science to improve autistic people's lives'. DOI: 10.5281/zenodo.3766923.   Ongoing development support for AutSPACEs platform by skilled intern.  
+Categorisation of impact	Cultural
+Formally Governed	Yes
+ 
+**Collaboration Titl**e	Fujitsu pro bono support 
+Partner
+**Organisation Name**	Fujitsu
+**Department**	Fujitsu Laboratories of Europe
+**Contributed Financially**	Yes
+**Contribution Currency**	GBP British Pound Sterling
+**Contribution Amount**	800
+**In-kind contribution**	Yes
+**In-kind contribution currency**	GBP British Pound Sterling
+**Give an estimate of the in-kind valu**e	333,000
+**Contributions Made**	We have provided Fujitsu with a significant amount of researcher time. 
+We have offered opportunities for development and upskilling for Fujitsu staff in open development, data management, GitHub, neurodiversity and participatory co-creation. 
+We have also given Fujitsu an opportunity to help meet their own charity partnership objective, and supported them to deliver a publiic presentation about their work for Autistica's 2020 Autism Research Festival. 
+**Partner Contributions**
+Fujitsu have provided a team of agile developers to work pro bono on the platform. 
+They have supported autistic people by remunerating them to take part in discussion sessions and user testing, as **well** as hosting these sessions. 
```

Comment:
renumerating

Reply:
it is remunerating ;) althuogh it seems like it should be renumerating becuase of numerical - such a confusing word!